src/Chainweb/PayloadProvider/EVM.hs

```diff
@@ -1296,7 +1294,7 @@ evmSyncToBlock p hints forkInfo = withLock (_evmLock p) $ do
             -- If we don't know that base, there's nothing we can do
             Nothing -> do
                 lf Warn $ "unknown base " <> brief rankedBaseHash
-                lf Info $ sshow forkInfo
+                lf Info $ encodeToText forkInfo
```

Comment:
When available, large data structures should be formatted as JSON instead of Haskell `Show` in the logs. This makes it easier to analyze the content by piping through `jq` or similar tools.